sharktank/sharktank/ops/qconv_impls.py

```diff
@@ -233,14 +238,14 @@ def _invoke_int32_conv2d(
 
 
 def _invoke_int32_pooling_sum(
-    input, kernel_size, stride, padding, dilation, *, accum_dtype
+    input, padded_input, kernel_size, stride, padding, dilation, *, accum_dtype
 ):
     """Invokes either a custom integer pooling sum or the built-in fp avg_pool2d
     kernel.
     """
     if debugging.flags.use_custom_int_conv_kernel:
         output = kernels.pooling_nchw_sum(
-            input,
+            padded_input,
```

Comment:
Same: please remove the padding function arg and the padding=padding from the fp call below.

Reply:
done